test/ut3_user/expectations/test_expectations_cursor.pkb

```diff
@@ -972,18 +972,16 @@ Rows: [ 4 differences ]
   procedure reports_on_exception_in_cursor
   as
     l_actual     sys_refcursor;
-    l_error_code integer := -19202; --Error occurred in XML processing
   begin
     --Act
     open l_actual for select 1/0 as error_column from dual connect by level < 10;
-    begin
       ut3.ut.expect(l_actual).to_be_empty();
-      --Assert
-      ut.fail('Expected '||l_error_code||' but nothing was raised');
-    exception
-      when others then
-        ut.expect(sqlcode).to_equal(l_error_code);
-    end;
+
+    ut.fail('Expected exception on cursor fetch');
+  exception
+    when others then
+      ut.expect(sqlerrm).to_be_like('%ORA-20218: SQL exception thrown when fetching data from cursor:%
+%ORA-01476: divisor is equal to zero%Check the query and data for errors%'); 
```

Comment:
Can you put the full error message in expected and use equal expectation?
It would improve readability for the change.

Reply:
done